frontend/src/services/getUserChats.ts

```diff
@@ -0,0 +1,3 @@
+import axios from 'axios';
+
+export const getUserChats = () => axios.get('https://randomuser.me/api/?results=20').then(res => res.data.results);
```

Comment:
I assume that this will be gone once we done with BE for chats?

Reply:
Sure thing